ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/documentation/fabric-documentation.j2

```diff
@@ -22,6 +22,12 @@
 {%         set fabric_switch.node = node %}
 {%         set fabric_switch.mgmt_ip = hostvars[node].switch_mgmt_ip | arista.avd.default('-') %}
 {%         do assigned_ip_addresses.append(hostvars[node].switch_mgmt_ip) if hostvars[node].switch_mgmt_ip is arista.avd.defined %}
+{%         for interface in hostvars[node].management_interfaces | arista.avd.natural_sort %}
+{%             if 'Management' not in interface %}
```

Comment:
I understand your method here, but I fear that it could fail too easily if someone use a shorter form of interface like Ma1 etc.
Could we test `if hostvars[node].management_interfaces[interface].ip_address is arista.avd.defined(fabric_switch.mgmt_ip)` instead, so we also get the bonus of variable check?

Reply:
Agree. Yours is a better option. Still, we can't control many things with this method